pkg/distro/fedora/distro.go

```diff
@@ -744,6 +796,33 @@ func newDistro(version int) distro.Distro {
 		minimalrawImgType,
 	)
 
+	if !common.VersionLessThan(rd.Releasever(), "38") {
+		// iot simplified installer was introduced in F38
+		x86_64.addImageTypes(
+			&platform.X86{
+				BasePlatform: platform.BasePlatform{
+					ImageFormat: platform.FORMAT_RAW,
+				},
+				BIOS:       false,
+				UEFIVendor: "fedora",
+			},
+			iotSimplifiedInstallerImgType,
+		)
+		aarch64.addImageTypes(
+			&platform.Aarch64{
+				BasePlatform: platform.BasePlatform{
+					FirmwarePackages: []string{
+						"uboot-images-armv8", // ??
+						"bcm283x-firmware",
+						"arm-image-installer", // ??
+					},
+				},
+				UEFIVendor: "fedora",
+			},
+			iotSimplifiedInstallerImgType,
```

Comment:
Do we want to enable the firmware overlays and /boot setup for the Pi here?

Reply:
I based these decisions entirely on the PR in composer that I moved over.  I'd ask @runcom.